docs/stable/6-payloads.md

```diff
@@ -14,8 +14,8 @@ title: 6/PAYLOADS
 
 ## Abstract
 
-This specifications decribes how the payload of each message in Status looks
-like. It is primarly centered around chat and chat-related use cases.
+This specifications describes how the payload of each message in Status looks
+like. It is primarily centered around chat and chat-related use cases.
 
 The payloads aims be flexible enough to support messaging but also cases
```

Comment:
I spotted this on while doing #114, if there are any other grammatical issues that you didn't cover already I'll address them here.

```suggestion
The payloads aim to be flexible enough to support messaging but also cases
```

Reply:
@Samyoul let's not do a all grammar issues here. I don't want this to get too large